src/sparseml/transformers/export.py

```diff
@@ -171,13 +171,15 @@ def export_transformer_to_onnx(
     model = load_task_model(task, model_path, config)
     _LOGGER.info(f"loaded model, config, and tokenizer from {model_path}")
 
+    model = model.train()
     trainer = Trainer(
         model=model,
         model_state_path=model_path,
         recipe=None,
         recipe_args=None,
         teacher=None,
     )
+    model = model.cpu()
```

Comment:
For some reason the model was getting sent to the gpu on my machine after the `trainer = Trainer` line above. I could not figure out what was doing it (I think its in the huggingface code).